### ISSUE TITLE
[Spell] Vindication proc chance

### ISSUE THREAD

**Issue by Dranah** [WoWManiaUK/Redemption#3829]
https://classic.wowhead.com/spell=26016/vindication

**What is Happening:**
The proc chance of the ability is acting weird. 
It seems to scale with how many points you put in it, 2/2 shows a way scarcer proc rate than 1/2. 
It also seems to scale with gear in some way (i suspect crit chance but i could be wrong), because ret paladins (and any test paladins geared as DPS) are getting more proc count, while protection ones (test paladins geared as tanks) get fewer, which is a controversy due to prot pallies having faster attack speed which should yield more frequent proc.
In general the debuff does not have the proper proc count and uptime, procs less often than intended.
Right now:
1) Having 1/2 talent grants you around 80-90% uptime as a DPS, and only about 50-60% as Tank.
2) Having 2/2 talent grants you around 20-30% uptime as a DPS, and only about 10-15% as Tank.
Most certainly the proc does not run on a ppm mechanic because you often get 2 procs within 3seconds after a minute or two having no procs at all.
In a raid situation, on a 2minute marrowgar fight I experience at average 5-6procs as a protection paladin, giving a 25-30% uptime, which is slightly higher than you get on dummies (possibly due to buffs and the effect scaling with something), but is far behind retail logs. See below.

**What Should happen:**
While I could not find an exact number on how much proc chance this debuff is supposed to have, based on retail footage and descriptions, on a single-target scenario it must have near 100% uptime and seldom ever drops off for a protection paladin. 
Here is a log from 3.3.5a on marrowgar on a 2minute parse the paladin had 33 procs of Vindication, which averages out to 1 proc every 4seconds and over 90% uptime:http://www.worldoflogs.com/reports/rt-91mjmt50my0vt6qc/details/17/?s=3154&e=3286
As you can see here: http://www.worldoflogs.com/reports/rt-91mjmt50my0vt6qc/details/17/?s=3154&e=3286 he had a total of 108 melee attacks capable of proc'ing the effect, so that calculates to be around 30% proc chance, give or take a few for RNG.
Comparing this log to my own parses of marrowgar on mania, the proc count and uptime is 6 times higher on retail.

TL;DR:
**How it works:** has barely 5% proc chance by default and scales with some stats. 2/2 gets less than half the procs of 1/2.
**How it should work:** should be around 30% proc chance for both 1/2 and 2/2, and not scale with anything

**Comment by Dranah** [comment]
There is no change.

**Comment by Keader** [comment]
Please test in ptr.
I tested and looks ok: https://youtu.be/q10fkUW_ISY
i will re-send to janitor ,the sql change

**Comment by Keader** [comment]
Hmm i send a new file to janitor.
I will closed it, because he will push to live tomorrow
